packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -848,36 +847,26 @@ export function findDownloadArtifactStep(transformationSteps: TransformationStep
     }
 }
 
-interface IDownloadResultArchiveParams {
-    jobId: string
-    downloadArtifactId: string
-    pathToArchive: string
-}
-export async function downloadResultArchive({
-    jobId,
-    downloadArtifactId,
-    pathToArchive,
-}: IDownloadResultArchiveParams) {
+export async function downloadResultArchive(
+    jobId: string,
+    downloadArtifactId: string | undefined,
+    pathToArchive: string,
+    downloadArtifactType: TransformationDownloadArtifactType
+) {
     let downloadErrorMessage = undefined
     const cwStreamingClient = await createCodeWhispererChatStreamingClient()
+
     try {
         await downloadExportResultArchive(
             cwStreamingClient,
             {
                 exportId: jobId,
                 exportIntent: ExportIntent.TRANSFORMATION,
-                exportContext: {
-                    transformationExportContext: {
-                        downloadArtifactId,
-                        downloadArtifactType: TransformationDownloadArtifactType.CLIENT_INSTRUCTIONS,
-                    },
-                },
             },
             pathToArchive
         )
     } catch (e: any) {
         downloadErrorMessage = (e as Error).message
-        // This allows the customer to retry the download
```

Comment:
Can users not retry now by clicking the blue download button / is that why we `throw e` below?

Reply:
The comment wasn't true (the code it was commenting wasn't responsible for allowing the customer to retry the download)